### PR TITLE
Updating CTD files and changing ctd script to account for oxygen sensor

### PR DESCRIPTION
**Pull request by funnyabc**

### CODE REVIEW

File: tools/cal_scripts/ctd_cal_parser.py [hunk 1]
```diff
@@ -56,6 +56,26 @@ def __init__(self):
             'T4': 'CC_T4',
             'T5': 'CC_T5',
         }
+        self.o_series_coefficients_map = {
```
Comment (danmergens) [inline tools/cal_scripts/ctd_cal_parser.py]: We split out the CTD and DO sensors (still need to finalize). Are we sure we want to pull in the oxygen sensor calibrations from the CTD here?

Comment (funnyabc) [reply tools/cal_scripts/ctd_cal_parser.py]: I thought we were leaving those in. Hence I put them in the CTD parser. What's the status of this?

File: calibration/CTDPFA/ATAPL-66662-00002__20170915.csv [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-16-7205,CC_ptcb2,0,
-16-7205,CC_ptempa0,-5.50755478e+001,
-16-7205,CC_ptempa1,5.58453062e+001,
-16-7205,CC_ptempa2,-5.87877286e-001,
-16-7205,CC_residual_temperature_correction_factor_a,-3.3433e-003,
-16-7205,CC_residual_temperature_correction_factor_b,1.5757e-004,
-16-7205,CC_residual_temperature_correction_factor_c,-2.3553e-006,
-16-7205,CC_residual_temperature_correction_factor_e,3.6000e-002,
-16-7205,CC_offset,0,
+serial,name,value,notes
```
Comment (danmergens) [inline calibration/CTDPFA/ATAPL-66662-00002__20170915.csv]: please avoid changing line endings

File: tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3098 (1).dev [hunk 3]
```diff
@@ -0,0 +1,19 @@
+ECO 	FLCDRTD-3098
```
Comment (danmergens) [inline tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3098 (1).dev]: duplicate file - remove?

File: tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3098 (2).dev [hunk 4]
```diff
@@ -0,0 +1,19 @@
+ECO 	FLCDRTD-3098
```
Comment (danmergens) [inline tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3098 (2).dev]: duplicate file - remove?

File: tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3716 (1).dev [hunk 5]
```diff
@@ -0,0 +1,19 @@
+ECO 	FLCDRTD-3716
```
Comment (danmergens) [inline tools/cal_scripts/FLCDRA/manufacturer_ARCHIVE/FLCDRTD-3716 (1).dev]: duplicate file - remove?

File: tools/cal_scripts/ctd_cal_parser.py (hunk 1 above)
Comment (danmergens) [reply tools/cal_scripts/ctd_cal_parser.py]: You're correct - we need to leave in place until the DO sensor is up and the CTD is using the new stream w/o DO values.

File: tools/cal_scripts/CTD/manufacturer_ARCHIVE/7248 (1).xmlcon [hunk 6]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SBE_InstrumentConfiguration SB_ConfigCTD_FileVersion="7.22.0.0" >
```
Comment (danmergens) [inline tools/cal_scripts/CTD/manufacturer_ARCHIVE/7248 (1).xmlcon]: duplicate file - remove

File: tools/cal_scripts/CTD/manufacturer_ARCHIVE/SBE19plusV2_50117 copy.xmlcon [hunk 7]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (danmergens) [inline tools/cal_scripts/CTD/manufacturer_ARCHIVE/SBE19plusV2_50117 copy.xmlcon]: duplicate file - remove

**Comment by danmergens** [comment]
You should also have a Git `.ignore` file for the `*.swp` files.

**Comment by funnyabc** [comment]
.swp files removed.

**Pull request merged**